terraform/ec2/win/main.tf

```diff
@@ -153,6 +153,7 @@ resource "null_resource" "integration_test_run" {
     user     = "Administrator"
     password = rsadecrypt(aws_instance.cwagent.password_data, local.private_key_content)
     host     = aws_instance.cwagent.public_dns
+    timeout = "5m"
```

Comment:
nit: we can remove this since 5m is the default?

Reply:
Good point, was tinkering with the timeouts before. Recent commit addresses that.